packages/vuetify/src/components/VAvatar/VAvatar.ts

```diff
@@ -40,6 +43,7 @@ export default mixins(
         minWidth: convertToUnit(this.size),
         width: convertToUnit(this.size),
         ...this.measurableStyles,
+        ...this.roundedClasses,
```

Comment:
This should go in `classes` computed, not `styles`.

Reply:
Ty, it wasn't clear for me